docs/sources/datasources/loki/query-editor/index.md

```diff
@@ -125,9 +125,9 @@ To re-order operations manually, drag the operation box by its name and drop it
 
 In same cases the query editor can detect which operations would be most appropriate for a selected log stream. In such cases it will show a hint next to the `+ Operations` button. Click on the hint to add the operations to your query.
 
-### Explain mode
+### Explain Query
```

Comment:
```suggestion
### Explain query
```

We should use sentence case capitalisation based on style guide: https://grafana.com/docs/writers-toolkit/style-guide/capitalization-punctuation/#sentence-case-capitalization